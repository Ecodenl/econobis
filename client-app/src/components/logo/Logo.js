import React from 'react';
import ImgLogo from '../../images/logo/logo-ECO-2025.png';

const Logo = props => {
    const style = {
        height: props.height,
        paddingTop: '10px',
        paddingBottom: '10px',
        width: 'auto',
    };

    return <img src={ImgLogo} className="img-logo" alt="Logo Eco" style={style} />;
};

export default Logo;
