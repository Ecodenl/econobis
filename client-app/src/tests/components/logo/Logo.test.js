import React from 'react';
import { shallow } from 'enzyme';

import Logo from '../../../components/logo/Logo';

test('should render Logo correctly', () => {
    const wrapper = shallow(<Logo height={'100px'} />);
    expect(wrapper).toMatchSnapshot();
});
