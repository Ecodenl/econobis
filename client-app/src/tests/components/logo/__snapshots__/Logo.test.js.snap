// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo correctly 1`] = `
<img
  alt="Logo Eco"
  className="img-logo"
  src={
    Object {
      "process": [Function],
    }
  }
  style={
    Object {
      "height": "100px",
      "paddingBottom": "10px",
      "paddingTop": "10px",
      "width": "auto",
    }
  }
/>
`;
